qa_priority_list/models.py

```diff
@@ -0,0 +1,77 @@
+import os
+import psycopg2
+
+DB_HOST = os.environ.get('DB_HOST')
+DB_NAME = os.environ.get('DB_NAME')
+DB_USER = os.environ.get('DB_USER')
+DB_PWD = os.environ.get('DB_PWD')
+LIMIT = os.environ.get('LIMIT', 2200)
+
+class Model:
+    def __init__(self):
+        self.conn = psycopg2.connect(host=DB_HOST, database=DB_NAME, user=DB_USER, password=DB_PWD)
+
+    def _execute_query(self, query):
+        data = []
+        cur = self.conn.cursor()
+        cur.execute()
+        if cur.rowcount == 0:
+            return []
+        row = cur.fetchone()
+        while row is not None:
+            data.append(row)
+            row = cur.fetchone()
+        return data
+
+
+class Projects(Model):
+
+    def get_projects_by_id_list(self, ids):
+        query = "SELECT * FROM organization_subscriptions where id in ({})".format(",".join(ids))
+        return self._execute_query(query)
+
+
+class Tickets(Model):
+
+    def get_tickets_by_project_id(self, project_id):
+        query = """
+        SELECT
+        Customer.email as Customer_email,
+        Project.title as Campain,
+        CONCAT(
+         'https://app.gigwalk.com/projects/',
+         Project.organization_id,
+         '/active/',
+         Project.id
+        ) as ProjectUrl,
+        CASE
```

Comment:
this logic doesn't seem right.   so IIRC, this deadline is the date, the ticket is submitted but not QA manual reviewed and the date that the system will auto-approve.  e.g. If I submit a ticket today, on 2.0, it seems after 14 days, if no manual approval, the system will approve the gig and pay me automatically.  So the idea is we want to have this deadline in place to remind QA going to check on the gig.